src/components/donate/Contact.js

```diff
@@ -1,7 +1,30 @@
 import React from "react";
 
 const Contact = () => {
-  return <div>Contact</div>;
+  return (
+    <div>
+      <p className="text-7xl">Contact Us</p>
```

Comment:
add font-urbanist 
check Figma for specific fonts and change fonts for all the following texts